### DISCUSSION TITLE
Support for ai commit description?

### DISCUSSION THREAD

**Discussion by eyyyyyyy3** [Zhengqbbb/cz-git#204] [Ideas]
Currently the ai only creates the commit description. What about also getting a description?

**Comment by Zhengqbbb** [comment]
```
<type>[optional scope]: <subject>

[optional body]

[optional footer(s)]
```

> What about also getting a description?

I guess the description here is the `body`?
If you are asking **whether AI can generate body content**: 
IMO AI cannot replace the writing of the body. If you need to add the body, it proves that you really need to convey something, and this matter requires genuine human writing. Therefore, czg only provides AI generate subject.

**Reply by eyyyyyyy3** [reply]
The AI could suggest some body or you could fine tune the output by providing keywords which the AI would then use in combination with the code to generate a body suggestion.
